### PR TITLE
reorg(web): integrated most-gestures into web package

### PR DESCRIPTION
**Pull request by z3dev**
These change integrate most-gestures into the web package, eliminating the need for the most-gesture package (maintained outside of JSCAD)

### All Submissions:

* [X] Have you followed the guidelines in our Contributing document?
* [X] Have you checked to ensure there aren't other open [Pull Requests](../../pulls) for the same update/change?
* [X] Does your submission pass tests?

### CODE REVIEW

**Comment by z3dev** [comment]
@SimonClark please review. i decided to bring most-gestures into the web package, as this makes maintenance easier. the code has already gone through changes due to lint/code standards.

File: packages/web/src/most-gestures/presses.js [hunk 1]
```diff
@@ -1,26 +1,19 @@
...
+const { exists } = require('./utils')
+
+/* alternative "clicks" (ie mouseDown -> mouseUp ) implementation, with more fine grained control */
+const basePresses = ({ mouseDowns$, mouseUps$, mouseMoves$, touchStarts$, touchEnds$, touchMoves$ }, settings) => {
+  touchMoves$ = touchMoves$.filter((t) => t.touches.length === 1)
 
   const starts$ = merge(mouseDowns$, touchStarts$) // mouse & touch interactions starts
   const ends$ = merge(mouseUps$, touchEnds$) // mouse & touch interactions ends
-  const moves$ = merge(mouseMoves$, touchMoves$)
+  // const moves$ = merge(mouseMoves$, touchMoves$)
```
Comment (mister-jscad) [inline packages/web/src/most-gestures/presses.js:10]: unused? then let's remove it.

**Review by mister-jscad — CHANGES_REQUESTED** [review]
Besides the requested change...

1. Add LICENSE from original package
2. Enable test script for web package

**Review by mister-jscad — APPROVED** [review]
OK. Good to go!

**Pull request merged**